.gitignore

```diff
@@ -2,7 +2,7 @@
 *.pyc
 .vscode
 __pycache__
-McPy/
+/McPy/
```

Comment:
Not entirely sure how gitignores work, but are you sure this isn't based on the root directory if a slash is preceding?

Reply:
/McPy/ is based on root directory but not McPy/

Reply:
Ah, so then it's wanted, but why though?

Reply:
When running startup.py file, it downloads the project and put it in a directory called McPy, this was probably why `McPy/` was in .gitignore
Here, because this line exists, it was ignoring directory at `/classes/McPy`.
I can remove the line if you want, for now if we don't launch the file the directory isn't created

Reply:
Done, I've removed this line